configure.ac

```diff
@@ -4973,6 +5014,15 @@ do
     ENABLED_SP_ECC=yes
     ENABLED_SP_EC_384=yes
     ;;
+  smallec1024 | smallp1024 | small1024)
```

Comment:
Might consider using small version for Linux Kernel Module or disabling...

Reply:
Moved the SAKKE and ECCSI to be included only when --enable-all or --enable-all-crypt and:
 ENABLED_FIPS = no
 ENABLED_LINUXKM_DEFAULTS != yes